### ISSUE TITLE
Unexpected name changes

### ISSUE THREAD

**Issue by marcel1334** [mi-erasmusmc/Tantalus#1]
I bumped into a strange "concept name changed". A letter "A" in the middle of a name was changed to "+é". Checked how often this occurs by searching for the character "é" and "ë". A lot of "concept name changed" records show up. Problem in Tantalus or actual changes in the vocab?

**Comment by PRijnbeek** [comment]
Yes i saw that was well. I think this might be related to loading the vocabs in the database, that encoding issue again. These are loaded in SQL Server in Azure Cloud. The new version has this issue need to check it in the source files.

It is not a Tantalus issue most probably, but it helps finding it.
